ext/crt.c

```diff
@@ -263,6 +263,8 @@ static PHP_GINIT_FUNCTION(awscrt) {
     awscrt_globals->log_level = 0;
 }
 
+#define CRT_VERSION "1.0.4"
+
```

Comment:
I recommend you use the normalized PHP_CRT_VERSION macro name, and define it in some header in package root directory, so this version will be checked by pecl during upload

See https://github.com/php/web-pecl/blob/master/public_html/release-upload.php#L140